style.css

```diff
@@ -46,7 +48,7 @@ ul {
 
 /* top container with title in */
 .container_top {
-    height: 100px;
+    height: 180px;
```

Comment:
You mentioned responsiveness the other day. The issue with the titles overflowing might be coming from the fact that this container has a fixed height. What happens if you don't explicitly define the height here and set overflow to auto?

Reply:
great thanks- this fixed my issue!